src/main/java/com/gregtechceu/gtceu/api/capability/recipe/FluidRecipeCapability.java

```diff
@@ -403,8 +403,8 @@ public void applyWidgetInfo(@NotNull Widget widget,
     // Maps fluids to a FluidEntryList for XEI: either a FluidTagList or a FluidStackList
     public static FluidEntryList mapFluid(FluidIngredient ingredient) {
         int amount;
-        if (ingredient instanceof IntProviderFluidIngredient provider) {
-            amount = provider.getCountProvider().getMaxValue();
+        if (ingredient instanceof IntProviderFluidIngredient) {
+            amount = 1;
```

Comment:
Doesn't this strip out actually needed info?

Reply:
we have a custom draw method for displaying the ranged output in EMI, its setting to 1 to not display an amount by default in emi